packages/nextjs/components/Processes/RecoveryProcess/RecoveryProcess.tsx

```diff
@@ -91,9 +106,9 @@ export const RecoveryProcess = ({
   if (recoveryStatus == RecoveryProcessStatus.SWITCH_TO_HACKED_ACCOUNT) {
     return (
       <CustomPortal
-        title={"Switch to hacked address"}
+        title={"Switch to Hacked Address"}
         description={
-          "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc vulputate libero et velit interdum, ac aliquet odio mattis. Class aptent taciti."
+          "To proceed with asset recovery, switch to the compromised address in your wallet. This step is essential to verify ownership and continue with the recovery process."
```

Comment:
"To proceed with asset recovery, switch to the compromised wallet. This step is essential to verify ownership and continue with the recovery process."